app-editors/leafpad/leafpad-0.8.18.1.ebuild

```diff
@@ -22,11 +23,10 @@ DEPEND="${RDEPEND}
 
 DOCS="AUTHORS ChangeLog NEWS README"
 
-src_prepare() {
-	epatch \
-		"${FILESDIR}"/${P}-fdo.patch \
-		"${FILESDIR}"/${P}-format-security.patch
-}
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.8.18.1-fdo.patch
```

Comment:
`P` to `PN` is an unnecessary change.

Reply:
Unneeded atm, sure, but  in the event this does get updated and the patch no longer applies it strikes me as simpler
to have `${PN}-version` already.